homework-2/routes/userRouter.ts

```diff
@@ -0,0 +1,45 @@
+import { Router } from 'express';
+import {
+    addUserHandler,
+    getAllUsersHandler,
+    getAutoSuggestHandler,
+    getUserByIdHandler,
+    removeUserHandler,
+    updateUserHandler
+} from '../middewares/userMiddlewares';
+import bodyParser from "body-parser";
+import { check } from 'express-validator';
+import UserService from '../services/userService';
+
+const PASSWORD_MATH_REGEXP = /^(?=.*[0-9])(?=.*[a-zA-Z])([a-zA-Z0-9]+)$/;
+
+const router: Router = Router();
+
+router.get('/auto-suggest', getAutoSuggestHandler);
+router.get('/', getAllUsersHandler);
+router.get('/:id', getUserByIdHandler);
+router.post(
+    '/',
+    bodyParser.json(),
+    check('login').exists().notEmpty().custom(UserService.validateLogin),
```

Comment:
I would try to unify validation functionality here

Reply:
unified in userValidator